apps/google-analytics-4/lambda/.env

```diff
@@ -4,7 +4,7 @@ SIGNING_SECRET: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 SENTRY_DSN: https://b8f524eae7c446fb8071476431426640@o2239.ingest.sentry.io/4504725335834624
 STAGE: test
 RELEASE_VERSION: test
-AWS_REGION: us-east-1
+DYNAMODB_REGION: us-east-1
```

Comment:
apparently AWS_REGION is a reserved keyword, so using a custom env var

Reply:
ahhh